frontend/awx/access/credential-types/CredentialTypePage/CredentialTypePage.tsx

```diff
@@ -23,14 +26,12 @@ export function CredentialTypePage() {
     data: credentialType,
     refresh,
   } = useGetItem<CredentialType>(awxAPI`/credential_types`, params.id);
-  const toolbarFilters = useCredentialTypesFilters();
-  const tableColumns = useCredentialTypesColumns();
-  const view = useAwxView<CredentialType>({
-    url: awxAPI`/credential_types/`,
-    toolbarFilters,
-    tableColumns,
+  const pageNavigate = usePageNavigate();
+  const actions = useCredentialTypeRowActions({
+    onCredentialTypesDeleted: () => pageNavigate(AwxRoute.CredentialTypes),
+    isDetailsPageAction: true,
```

Comment:
Looks like this prop isn't being used anywhere in the hook. We can remove it and just pass a single prop here.